pkg/ovs/openflow/ofctrl_builder.go

```diff
@@ -634,14 +640,18 @@ func (b *ofFlowBuilder) MatchCTDstPort(port uint16) FlowBuilder {
 // "+new", "+est", "+rel" and "+trk-inv".
 func (b *ofFlowBuilder) MatchCTProtocol(proto Protocol) FlowBuilder {
 	switch proto {
-	case ProtocolTCP:
+	case ProtocolTCP, ProtocolTCPv6:
 		b.Match.CtIpProto = 6
-	case ProtocolUDP:
+	case ProtocolUDP, ProtocolUDPv6:
 		b.Match.CtIpProto = 17
-	case ProtocolSCTP:
+	case ProtocolSCTP, ProtocolSCTPv6:
 		b.Match.CtIpProto = 132
 	case ProtocolICMP:
 		b.Match.CtIpProto = 1
+	case ProtocolICMPv6:
+		b.Match.CtIpProto = 58
+	case ProtocolIGMP:
```

Comment:
IGMP packet does not enter conntrack. We do not use conntrack for multicast.

Reply:
done